mender-update/inventory.cpp

```diff
@@ -105,11 +110,16 @@ error::Error PushInventoryData(
 	// replace the trailing comma with the closing square bracket
 	payload[payload.size() - 1] = ']';
 
+	size_t payload_hash = std::hash<string> {}(payload);
```

Comment:
Had to look this one up, so we call a function on the constructed object to hash..

Reply:
Yeah, it's a bit confusing.